src/ui/components/address-action/ApplicationLine/ApplicationLine.tsx

```diff
@@ -1,22 +1,23 @@
-import React, { useLayoutEffect, useState } from 'react';
 import { animated, useTransition } from '@react-spring/web';
 import { capitalize } from 'capitalize-ts';
+import { ethers } from 'ethers';
```

Comment:
The only significant change here is the` import { ethers } from 'ethers';`. The other changes are related to using the "organize imports" code action (which, by the way, does a great job of organizing the imports)

Reply:
Ideally library imports should be above project imports, but for some reason this doesn't work

Reply:
it is not that smart yet to tell the difference between icon or library imports, updated